pkgs/applications/science/misc/root/default.nix

```diff
@@ -20,6 +20,13 @@ stdenv.mkDerivation rec {
 
     # this prevents thisroot.sh from setting $p, which interferes with stdenv setup
     ./thisroot.patch
+
+    # https://sft.its.cern.ch/jira/browse/ROOT-8728
+    (fetchpatch {
+      url = "https://sft.its.cern.ch/jira/secure/attachment/20025/0001-std-string_view-has-no-more-to_string.patch";
+      sha256 = "0ngyk960xfrcsj4vhr1ax8h85fx0g1cfycxi3k35a6ych2zmyg8q";
+    })
+    ./ROOT-8728-extra.patch
```

Comment:
No, this one I had to make myself.

There is a certain commit in root-project/root#522 which might have a similar effect. But that commit is not permanently stored on github and can be removed at any moment. Also their approach is to mark ```operator std::string_view()``` as ```explicit``` which might or might not be accepted. The way I do my patching will force user to do explicit conversions, which should be always forward-compatible with future possible upstream. This is because it will not allow to write code that won't be compatible if the ```explicit```-based solution is not accepted.

If you want, I can publish my patch as a gist and never delete it. Or I can try to put it on their bugtracker, I hope they won't mind :)